### ISSUE TITLE
False positives from loadInclude rule

### ISSUE THREAD

**Issue by alexpott** [mglaman/phpstan-drupal#516]
Code like
```
  $module_handler = \Drupal::moduleHandler();
  $enabled_modules = $module_handler->getModuleList();

  foreach (array_keys($enabled_modules) as $module) {
    // All modules should have a recorded schema version, but when they
    // don't, detect and fix the problem.
    if (!isset($versions[$module])) {
      // Ensure the .install file is loaded.
      $module_handler->loadInclude($module, 'install');
      $all_updates = $update_registry->getAvailableUpdates($module);
```
(see update.inc)

results in 
```
		-
			message: "#^A file could not be loaded from Drupal\\\\Core\\\\Extension\\\\ModuleHandlerInterface\\:\\:loadInclude$#"
			count: 1
			path: includes/update.inc
```
which it should not. There are instances where the first argument to loadInclude is a variable and in these instance PHPStan should not result in an error.
